draft-birkholz-cose-receipts-ccf-profile.md

```diff
@@ -47,7 +47,7 @@ author:
 
 normative:
   RFC9162: certificate-transparency-v2
-  I-D.ietf-cose-merkle-tree-proofs: cose-receipts
+  I-D.draft-ietf-cose-merkle-tree-proofs: cose-receipts
```

Comment:
I think this is right, but I am not sure how to check that references work properly. I am willing to learn though.